tools/WinObjC.Packaging/WinObjC.Packageable.targets

```diff
@@ -0,0 +1,88 @@
+﻿<?xml version="1.0" encoding="utf-8" standalone="no"?>
+
+<!--
+  PACKAGABLE TARGETS
+  Description: The target definitions specific to a WinObjC project intended to be packaged
+  For Use By: A VcxProj that can be packaged into a NuGet
+-->
+
+<Project xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
+  <PropertyGroup>
+    <TargetOsAndVersion Condition="'$(TargetOsAndVersion)' == ''">Universal Windows</TargetOsAndVersion>
+  </PropertyGroup>
+
+  <!-- Presets for a file's location in the package -->
+  <PropertyGroup>
+    <ShouldAssignPackageContentBasePath Condition="'$(PackageContentBasePath)' == ''">true</ShouldAssignPackageContentBasePath>
+    <HasConfigurationSpecificPackageContents Condition="'$(HasConfigurationSpecificPackageContents)' == ''">false</HasConfigurationSpecificPackageContents>
+    <PackageContentBasePath Condition="'$(ShouldAssignPackageContentBasePath)' == 'true'">build\lib\$(TargetOsAndVersion)\$(PlatformTarget)</PackageContentBasePath>
+    <PackageContentBasePath Condition="'$(ShouldAssignPackageContentBasePath)' == 'true' and '$(HasConfigurationSpecificPackageContents)' == 'true'">$(PackageContentBasePath)\$(Configuration)</PackageContentBasePath>
+
+    <ShouldAssignPackageHeaderBasePath Condition="'$(PackageHeaderBasePath)' == ''">true</ShouldAssignPackageHeaderBasePath>
+    <PackageHeaderBasePath Condition="'$(ShouldAssignPackageHeaderBasePath)' == 'true'">build\include\$(MSBuildProjectName)</PackageHeaderBasePath>
+
+    <ShouldAssignPackageResourcesBasePath Condition="'$(PackageHeaderBasePath)' == ''">true</ShouldAssignPackageResourcesBasePath>
+    <PackageResourcesBasePath Condition="'$(ShouldAssignPackageHeaderBasePath)' == 'true'">build\content\$(MSBuildProjectName)</PackageResourcesBasePath>
+
+    <ShouldAssignPackageBundleBasePath Condition="'$(PackageBundleBasePath)' == ''">true</ShouldAssignPackageBundleBasePath>
+    <PackageBundleBasePath Condition="'$(ShouldAssignPackageBundleBasePath)' == 'true'">build\bundle\$(MSBuildProjectName)</PackageBundleBasePath>
+  </PropertyGroup>
+
+  <PropertyGroup>
+    <GetPackageContentsDependsOn>$(GetPackageContentsDependsOn);$(ComputeLinkInputsTargets);FindPublicHeaders</GetPackageContentsDependsOn>
+  </PropertyGroup>
+
+  <!-- Add MSBuild outputs into the package -->
+  <Target Name="AddBuiltOutput" BeforeTargets="GetPackageContents"  DependsOnTargets="$(GetPackageContentsDependsOn)" Condition="'$(PresetPackageContents)' == 'true'">
+    <!-- Framework / Static Library Projects -->
+    <ItemGroup Condition="'$(ConfigurationType)' == 'StaticLibrary'">
+      <!-- .lib Files -->
+      <_ImportLibraryFileNames Include="$(OutDir)*.lib" />
```

Comment:
>OutDir [](start = 42, length = 6)

Is this what we want? I was thinking that TargetPath was the more generalized thing or whatever I did for the frameworks ones ...

      <_ImportLibraryFileNames Include="@(Link -> '%(ImportLibrary)')">
      </_ImportLibraryFileNames>


      <PackageFile Include="@(_ImportLibraryFileNames->Distinct())" Condition="Exists('%(FullPath)')">
        <PackagePath>$(PackageContentBasePath)\%(Filename)%(Extension)</PackagePath>
      </PackageFile>


The ImportLibrary metadata on Link is telling it what to make the importLibrary called / where to output it so I think that is the safest? spot? I think OutDir*.lib is going to maybe pick up more than you want if other packages etc dump stuff to the output folder.